app-text/sigil/sigil-1.2.1-r1.ebuild

```diff
@@ -53,6 +53,8 @@ S="${WORKDIR}/Sigil-${PV}"
 
 DOCS=( ChangeLog.txt README.md )
 
+PATCHES=( "${FILESDIR}/sigil-qt5.15-fix.patch" )
```

Comment:
we usually use `${P}` prefix (first ebuild version to use that patch) instead of `${PN}`.

Reply:
fixed